src/main/java/gov/usds/case_issues/controllers/TagFragment.java

```diff
@@ -0,0 +1,29 @@
+package gov.usds.case_issues.controllers;
```

Comment:
This should probably move to a free-standing package with other constraints, if/when we create such things, so that we can use the same one for the DB Models without creating circular package dependencies (`constraints`? `validation`? Probably the latter). Right now there's just one, so this seems like an OK place for it to live.